pyanaconda/core/system.py

```diff
@@ -0,0 +1,53 @@
+#
+# system.py:  Provide information about system which runs Anaconda
```

Comment:
It should have been live_system but this seems to me better as it could be a place for functions working with the current installer environment. Any better idea?

Reply:
I wanted to have it more generic but truth is it could be renamed later

Reply:
Renamed.